src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,135 @@
+package task;
+
+import duke.UI;
+import java.util.ArrayList;
+
+/**
+ * Represents a list housing all the Tasks.
+ */
+public class TaskList {
+
+    /**
+     * List to house the Tasks.
+     */
+    private ArrayList<Task> list;
```

Comment:
Maybe plural form should be used on names representing a collection of objects? e.g. tasks